setup.py

```diff
@@ -194,7 +194,7 @@ def extract_version():
 
 
 setup(
-    name='Iris',
+    name='iris-scitools',
```

Comment:
there appears to be something of a L:R split here

I think that the practical aspect of `iris` appearing at the start of the string should carry the day, e.g.

> If the packages were sorted alphabetically, and someone looked for iris they'd find iris-scitools nearby.

Although `scitools-iris` might feel more natural to some developers, I think for the 'i want to find a package' use case which pypi addresses, `iris-scitools` appears more practical to me